rofi-pass

```diff
@@ -405,7 +405,7 @@ typeMenu () {
   if [[ -n $default_do ]]; then
     if [[ $default_do == "menu" ]]; then
       checkIfPass
-      typefield=$(printf '%s\n' "${!stuff[@]}" | sort | _rofi -dmenu  -p "Choose Field to type > ")
+      typefield=$({ echo autotype ; printf '%s\n' "${!stuff[@]}" | grep -v '^autotype$' | sort; } | _rofi -dmenu  -p "Choose Field to type > ")
```

Comment:
The exception makes sense, but could possibly use bash features instead of using `grep`?!
Not sure if it is worth it.
It should use "${AUTOTYPE_field}" here however anyway.

Reply:
You could also set `default_do=autotype` I guess?
And then use `alt-t` to get to the typeMenu if you need it.